src/components/file-selector/file-selector.ts

```diff
@@ -4,7 +4,7 @@ import { customElement, property, state } from 'lit/decorators.js';
 import { ref } from 'lit/directives/ref.js';
 import { html, unsafeStatic } from 'lit/static-html.js';
 
-import type { SbbButtonElement } from '../button';
+import { type SbbSecondaryButtonStaticElement } from '../button/secondary-button-static';
```

Comment:
Please also use direct import to secondary-button-static insteaf of '../button';
```suggestion
import type { SbbSecondaryButtonStaticElement } from '../button/secondary-button-static';
```